src/main/java/com/monumental/models/Monument.java

```diff
@@ -78,16 +75,12 @@ public Monument() {
     }
 
     public Monument(String artist, String title, Date date, String city, String state) {
+        this();
```

Comment:
What does this do?

Reply:
Calls the default constructor with no params

Reply:
duh